Server/src/tests/unit/fileUploadController.test.js

```diff
@@ -0,0 +1,25 @@
+const fileUploadController = require("../../controllers/fileUploadController");
+const dataUploadModel = require("../../models/DataUploadModel");
+
+dataUploadModel.insertReferenceType = jest.fn();
+
+//check if there is createRequest function
+describe("fileUploadController.createRequest function", ()=>{
```

Comment:
These will likely need to get modified after the TS conversion. It wont be much but will require some code changes. @Mehrdadmaskull